netket/graph/lattice.py

```diff
@@ -19,11 +19,12 @@
 import itertools
 import networkx as _nx
 import warnings
+from typing import Tuple, Union, Optional
 
 
-def get_edges(atoms_positions, cutoff):
+def get_edges(atoms_positions, cutoff, tol=1e-5):
```

Comment:
We already discussed this in the other PR.
Can you use this code in here?
https://github.com/netket/netket/blob/17035c8b8b89c401d49acd0b743d8383bef0d0db/netket/graph/lattice.py#L23